notebooks/lesson05_random-forest-deep-dive.ipynb

```diff
@@ -0,0 +1,1577 @@
+{
```

Comment:
<p>maybe this list comprehension looks a bit like magic to the students with less python experience. to explain you could maybe add the same code with a for-loop below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>good idea. done</p>